pkg/overlay/service.go

```diff
@@ -28,13 +31,16 @@ var (
 	options                                                                                  peertls.TLSFileOptions
 )
 
+var home, _ = homedir.Dir()
+var boltDBPath = filepath.Join(home, ".storj", "overlaydb.db")
```

Comment:
these should probably be in an init method. i don't know if the go language spec guarantees that these two variables are defined in order

Reply:
like

```golang
var (
  home, boltDBPath string
)

func init() {
  home, _ = homedir.Dir()
  boltDBPath = filepath.Join(...)
}
```

perhaps even better

```
func defaultBoltDBPath() string {
   ...
}
```
no global variables, and just use the return value in the call to define the flag

Reply:
Done.